draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -226,39 +225,39 @@ The Holder can then create a signed document, the Key Binding JWT as defined in
 data provided by the Verifier such as a nonce to ensure the freshness of the signature, and audience to indicate the
 intended audience for the document.
 
-The Key Binding JWT can be included in the Combined Format for Presentation and sent to the Verifier along with the SD-JWT and the Holder-Selected Disclosures.
+The Key Binding JWT can be included as part of the SD-JWT and sent to the Verifier as described in (#sd-jwt-structure).
 
-Note that there may be other ways to send a Key Binding JWT to the Verifier or for the Holder to prove possession of the key material included in an SD-JWT. In these cases, inclusion of the Key Binding JWT in the Combined Format for Presentation is not required.
+Note that there may be other ways to send a Key Binding JWT to the Verifier or for the Holder to prove possession of the key material included in an SD-JWT. In these cases, inclusion of the Key Binding JWT in the SD-JWT is not required.
 
 ## Verification
 
 At a high level, the Verifier
 
- * receives the Combined Format for Presentation from the Holder and verifies the signature of the SD-JWT using the Issuer's public key,
+ * receives the SD-JWT from the Holder and verifies its signature using the Issuer's public key,
  * verifies the Key Binding JWT, if Key Binding is required by the Verifier's policy, using the public key included in the SD-JWT,
  * calculates the digests over the Holder-Selected Disclosures and verifies that each digest is contained in the SD-JWT.
 
 The detailed algorithm is described in (#verifier_verification).
 
 # Data Formats {#data_formats}
 
-This section defines data formats for SD-JWTs, Disclosures, Key Binding JWTs and formats for combining these elements for transport.
+This section defines data formats for SD-JWTs including the Issuer-signed JWT content, Disclosures, and Key Binding JWTs.
 
-## Format of an SD-JWT {#creating_sd_jwt}
+## SD-JWT Payload
```

Comment:
```suggestion
## Issuer-signed JWT Payload
```
Wouldn't this be more accurate..?

Reply:
I think they are basically synonymous. And prefer SD-JWT Payload here.